src/styles/style.css

```diff
@@ -0,0 +1,141 @@
+.page {
+  font-family: Poppins, Roboto, sans-serif;
+  color: #fff;
+  font-size: 16px;
+  line-height: 140%;
+
+  scroll-behavior: smooth;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.page__menu {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+
+  opacity: 0;
+  pointer-events: none;
+}
+
+.page__menu:target {
+  opacity: 1;
+  pointer-events: all;
+}
+
+.header {
+  box-sizing: border-box;
+  height: 100vh;
+  background-image: url(../images/header-bg-mobile.png);
+  background-size: cover;
+  background-position: right;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.header__title {
+  margin: 0;
+  text-align: center;
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 100%;
+  letter-spacing: -1.5px;
+  padding: 0 20px 88px;
+}
+
+.top-bar {
+  padding: 24px 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.top-bar__icons {
+  display: flex;
+  gap: 24px;
+}
+
+.top-bar__logo {
+  display: flex;
+}
+
+.menu {
+  box-sizing: border-box;
+  height: 100vh;
+  background-color: #C2C2C2;
+  font-weight: 600;
+  text-transform: uppercase;
+}
+
+.menu__top {
+  margin-bottom: 24px;
+}
+
+.menu__nav {
+  margin-bottom: 40px;
+}
+
+.menu__content {
+  padding-inline: 20px;
+}
+
+.menu__phone--number {
+  display: block;
+  color: #1D1D1D;
+  text-decoration: none;
+  font-size: 16px;
+  line-height: 27px;
+  margin-bottom: 16px;
+}
+
+.menu__book--ride {
+  display: block;
+  color: #1D1D1D;
+  text-decoration: none;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 2px;
+  padding-bottom: 7px;
+  border-bottom: 1px solid #1D1D1D;
+  width: fit-content;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+}
+
+.nav__link {
+  color: #1D1D1D;
+  text-decoration: none;
```

Comment:
the same with `text-decoration: none;`

you can delete this style in your scss blocks for links because you reseted it in your reset.scss